components/WhatsNew.vue

```diff
@@ -62,21 +79,35 @@ export default {
   @include card-container();
   padding: 10px;
   margin-bottom: 20px;
+  width: 100%;
 }
 
 .WhatsNew-heading {
   display: flex;
   align-items: center;
   @include card-h2();
   margin-bottom: 12px;
-  color: $gray-2;
+  // color: $gray-2;
   margin-left: 12px;
-
+  height: 28px;
   &-icon {
     margin: 3px;
   }
 }
 
+.WhatsNew-subheading {
+  font-size: 16px;
+  font-weight: bold;
+  margin-left: 12px;
+  display: flex;
+  width: 100%;
+}
+
+.WhatsNew-subheading p {
+  width: 9%;
```

Comment:
this is creating a misalignment with your header and content. Setting the first p child to a static width would prob fix it. `p:first-child: 130px;`

<img width="589" alt="Screen Shot 2020-09-18 at 11 54 24 AM" src="https://user-images.githubusercontent.com/5452152/93634883-2fcbe380-f9a6-11ea-90c1-4882ae7a8380.png">

Reply:
addressed in latest commit